ui/components/app/flask/experimental-area/index.scss

```
@@ -0,0 +1,44 @@
+.experimental-area {
+  color: $flask-purple;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .logo {
+    padding: 16px;
+    line-height: 0.625rem;
+    font-family: monospace;
+  }
```

This reduces the size, uses design system sass variable and adds more space below the logo

```suggestion
  .logo {
    padding: 16px 8px 0;
    line-height: 0.625em;
    font-family: monospace;
    font-size: $font-size-h8;
    margin-bottom: 32px;
  }
```